src/components/About/About.module.css

```diff
@@ -0,0 +1,4 @@
+.title {
+    text-align: center;
+  } 
```

Comment:
Codacy found an issue: [Expected indentation of 0 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/todo_test/pullRequest?prid=5750332)